packages/cli/src/commands/doctor/versionRanges.ts

```diff
@@ -4,6 +4,7 @@ export default {
   YARN: '>= 1.10.x',
   NPM: '>= 4.x',
   WATCHMAN: '4.x',
+  PYTHON: '>= 2.x', // Is there a way to force it <= 3 ?
```

Comment:
Yes, you can use `>= 2.x <= 3` for example. This is passed straight to `semver.satisfies` so anything semver-valid is good